finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,31 @@
+//tenSpeckledFrogs.js
+
+
+//  ## Required  and ## Extension 1
+
+function frogRhyme(frogNumber) {
```

Comment:
@Afoucheaux - I think you're missing one case in your conditional here. While testing out the code, I noticed the following print out incorrectly:

> 2 speckled frogs sat on a log eating some most delicious bugs. One jumped in the pool where its nice and cool then **there were 1 speckled frogs**.

How could you change this to fix that issue?